src/components/ChatContent/DiffContent.tsx

```diff
@@ -18,12 +18,26 @@ import {
 type DiffType = "apply" | "unapply" | "error" | "can not apply";
 
 function toDiff(str: string): string {
-  const replaceEscapedEOL = str
-    .split("\n")
-    .filter((_) => _)
```

Comment:
I think the issue was from `filter` removing empty lines.

What's the `\\` part for?

Reply:
`\n` won't match `\\n` ;)